library/src/main/java/io/github/boguszpawlowski/composecalendar/day/DayState.kt

```diff
@@ -13,4 +14,5 @@ public typealias NonSelectableDayState = DayState<EmptySelectionState>
 public data class DayState<T : SelectionState>(
   private val day: Day,
   val selectionState: T,
+  val eventState: EventState
```

Comment:
I believe adding such functionality shouldn't be in scope of this PR.

Reply:
In my project, I display the events of the day as dots in the calendar. Why not add this functionality to this PR

Reply:
Because this is easily achievable via the existing library API, and adding additional parameters to the API increases its complexity and reduces readability. Also, not every project will want to represent events as dots.